task/power/src/main.rs

```diff
@@ -179,6 +180,25 @@ fn controllers() -> [PowerController; 10] {
     ]
 }
 
+#[cfg(target_board = "gimlet-b")]
+fn controllers() -> [PowerController; 11] {
+    let task = I2C.get_task_id();
+
+    [
+        rail_controller!(task, IBC, bmr491, v12_sys_a2, A2),
```

Comment:
Should the DRAM Vpp ISL68224 be here? This is U352 with outputs for:

* VPP_ABCD_A0_REG
* VPP_EFGH_A0_REG
* V1P8_SP3_VDD_18_A0_REG
* 
In addition, there's the NIC TPS546B24 -- U565; V0P96_NIC_VDD_A0HP.

Reply:
Yes -- will add those on the Gimlet A side as well.

Reply:
Done